examples/interactions/commands/application_commands.py

```diff
@@ -59,12 +59,14 @@ async def favorite(
     # and if it matches up to an anime,
     # it responds with a custom response.
     anime = anime.value
-    
+
     match anime:
-        case "Attack on Titan":
+        case 'Attack on Titan':
```

Comment:
Reverts out of scope change.
```suggestion
        case "Attack on Titan":
```